tests/dummy/app/styles/app.scss

```diff
@@ -176,4 +176,34 @@
   }
 }
 
+.frost-list-demo-primitives {
```

Comment:
I don't want to see anything used in the list item coming from the dummy - if there is value in flex primitives then it should be part of the consumable interface